### PR TITLE
fix scheduler policy issue

### PR DESCRIPTION
**Pull request by weekface**
I had tested in kubernetes v1.10, v1.11 and v1.12

@tennix @onlymellb @xiaojingchen PTAL

### CODE REVIEW

File: charts/tidb-operator/values.yaml [hunk 1]
```diff
@@ -50,5 +50,5 @@ scheduler:
     requests:
       cpu: 80m
       memory: 50Mi
-  # modify the hyperkube verison to your kubernetes version
-  kubeSchedulerImage: gcr.io/google-containers/hyperkube:v1.10.12
+  # this hyperkube verison must be the same as your kubernetes cluster version
+  kubeSchedulerImage: gcr.io/google-containers/hyperkube:v1.12.1
```
Comment (tennix) [inline charts/tidb-operator/values.yaml]: Comment this out and make this field `required`, so users have to fill this out when installing tidb-operator.

File: docs/google-kubernetes-tutorial.md [hunk 2]
```diff
@@ -93,9 +93,10 @@ When you see `Running`, it's time to hit `Control + C` and proceed to the next s
 
 The first TiDB component we are going to install is the TiDB Operator, using a Helm Chart. TiDB Operator is the management system that works with Kubernetes to bootstrap your TiDB cluster and keep it running. This step assumes you are in the `tidb-operator` working directory:
 
+	KUBE_VERSION=$(kubectl version --short | awk '/Server/{print $3}' | awk -F '-' '{print $1}') &&
```
Comment (weekface) [inline docs/google-kubernetes-tutorial.md]:
```suggestion
	KUBE_VERSION=$(kubectl version --short | awk '/Server/{print $NF}' | awk -F '-' '{print $1}') &&
```

**Pull request merged**